sys-process/cpu-x/cpu-x-3.2.4.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg
+
+MY_PN='CPU-X'
+DESCRIPTION='A Free software that gathers information on CPU, motherboard and more'
+HOMEPAGE="https://x0rg.github.io/${PN}"
+SRC_URI="https://github.com/X0rg/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE='GPL-3+'
+SLOT='0'
+KEYWORDS='-* ~amd64'
+IUSE='l10n_cs l10n_fr l10n_id l10n_pl l10n_pt-BR l10n_ru l10n_zh
++gtk +ncurses static-libs
++bandwidth check-update +dmidecode force-libstatgrab portable'
+RESTRICT='primaryuri'
+PATCHES=(
+	"${FILESDIR}/${P}-check-update.patch"
+	"${FILESDIR}/${P}-l10n-support.patch"
+	"${FILESDIR}/${P}-static-libs-fix.patch"
+)
+S="${WORKDIR}/${MY_PN}-${PV}"
+
+DEPEND='
+		gtk? ( >=x11-libs/gtk+-3.12:3 )
+		ncurses? ( sys-libs/ncurses )
+		sys-apps/pciutils
+		>=sys-libs/libcpuid-0.3.0
+		check-update? ( dev-libs/json-c net-misc/curl )
+		!force-libstatgrab? ( sys-process/procps )
+		force-libstatgrab? ( sys-libs/libstatgrab )'
+
+BDEPEND='dev-lang/nasm'
+
+src_prepare() {
```

Comment:
This should be the default function that's called. Ebuild should work without `src_prepare` defined here at all.

Reply:
I did this because the eclass **xdg** has also a _src_prepare_ function.

Reply:
You are correct :) I failed to see it was xdg rather than xdg-utils. It's good work here.

I tested these and they build and cpu-x runs just fine. I still see some styling issues in the ebuilds, but can confirm they work as intended.